roles/download/defaults/main.yml

```diff
@@ -171,8 +171,7 @@ kubedns_image_tag: "{{ kubedns_version }}"
 
 coredns_version: "1.2.5"
 coredns_image_repo: "coredns/coredns"
-coredns_image_tag: "{{ coredns_version }}{%- if image_arch != 'amd64' -%}__{{ image_arch}}_linux{%- endif -%}"
-
+coredns_image_tag: "{{ coredns_version }}"
```

Comment:
I think this is wrong and will break.

I guess this would be more correct:

`{%- if image_arch != 'amd64' -%}coredns-{{ image_arch}}{%- else -%}{{ coredns_version }}{%- endif -%}`

Reply:
https://hub.docker.com/r/coredns/coredns/tags/

Reply:
Wow yeah what the 🦀 

Will update ASAP, I seem to have forgotten to push the latest change 😞

Reply:
For more information see:
https://blog.docker.com/2017/09/docker-official-images-now-multi-platform/

I am looking into a way to prove that the tags are multiplatform through the docker API.